x-pack/plugins/security_solution/cypress/ccs_integration/detection_rules/event_correlation_rule.spec.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { esArchiverCCSLoad, esArchiverCCSUnload } from '../../tasks/es_archiver';
+import { formatMitreAttackDescription } from '../../helpers/rules';
+import { getEqlRule, getCCSEqlRule, getIndexPatterns } from '../../objects/rule';
+
+import { ALERT_DATA_GRID, NUMBER_OF_ALERTS } from '../../screens/alerts';
+import {
+  CUSTOM_RULES_BTN,
+  RISK_SCORE,
+  RULE_NAME,
+  RULES_ROW,
+  RULES_TABLE,
+  RULE_SWITCH,
+  SEVERITY,
+} from '../../screens/alerts_detection_rules';
+import {
+  ABOUT_DETAILS,
+  ABOUT_INVESTIGATION_NOTES,
+  ABOUT_RULE_DESCRIPTION,
+  ADDITIONAL_LOOK_BACK_DETAILS,
+  CUSTOM_QUERY_DETAILS,
+  DEFINITION_DETAILS,
+  FALSE_POSITIVES_DETAILS,
+  getDetails,
+  removeExternalLinkText,
+  INDEX_PATTERNS_DETAILS,
+  INVESTIGATION_NOTES_MARKDOWN,
+  INVESTIGATION_NOTES_TOGGLE,
+  MITRE_ATTACK_DETAILS,
+  REFERENCE_URLS_DETAILS,
+  RISK_SCORE_DETAILS,
+  RULE_NAME_HEADER,
+  RULE_TYPE_DETAILS,
+  RUNS_EVERY_DETAILS,
+  SCHEDULE_DETAILS,
+  SEVERITY_DETAILS,
+  TAGS_DETAILS,
+  TIMELINE_TEMPLATE_DETAILS,
+} from '../../screens/rule_details';
+
+import {
+  goToManageAlertsDetectionRules,
+  waitForAlertsIndexToBeCreated,
+  waitForAlertsPanelToBeLoaded,
+} from '../../tasks/alerts';
+import {
+  changeRowsPerPageTo100,
+  filterByCustomRules,
+  goToCreateNewRule,
+  goToRuleDetails,
+  waitForRulesTableToBeLoaded,
+} from '../../tasks/alerts_detection_rules';
+import { createTimeline } from '../../tasks/api_calls/timelines';
+import { createEventCorrelationRule } from '../../tasks/api_calls/rules';
+import { cleanKibana } from '../../tasks/common';
+import {
+  createAndActivateRule,
+  fillAboutRuleAndContinue,
+  fillDefineEqlRuleAndContinue,
+  fillScheduleRuleAndContinue,
+  selectEqlRuleType,
+  waitForAlertsToPopulate,
+  waitForTheRuleToBeExecuted,
+} from '../../tasks/create_new_rule';
+import { loginAndWaitForPageWithoutDateRange } from '../../tasks/login';
+
+import { ALERTS_URL } from '../../urls/navigation';
+
+describe('Detection rules', function () {
+  const expectedNumberOfRules = 1;
+  const expectedNumberOfAlerts = '100 alerts';
+
+  beforeEach(function () {
+    cleanKibana();
+    esArchiverCCSLoad('run-parts');
```

Comment:
Is there any reason why the name of the es_archive file is that one?  I think it is better if we can use a more revealing name for the archive and we follow the snake case notation as we do with the others :)

Reply:
The archive contains some automatic activities performed by a Debian 10 system. It uses the command `run-parts`, hence the name. I think it can be used to check a few things like process events, sequences, parent-child relations.

I'm not particularly attached to the name or the casing, feel free to suggest alternatives.

Reply:
Also, I saw that the file is huge. We are trying to minimize the size of these types of files. Can we try to minimize it to have just 2 or 3 alerts? We are also committing now the data.json file directly without compressing it. This is to simplify the debug process if something fails.

Reply:
About the naming, what do you think about something like `debian_10_run_parts`?